app/controllers/application_controller.rb

```diff
@@ -1,2 +1,13 @@
 class ApplicationController < ActionController::Base
+protect_from_forgery with: :exception
+
+before_action :configure_permitted_parameters, if: :devise_controller?
+
+protected
+
+def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:email])
```

Comment:
I'm not sure if we need to add email as a key, isn't email already included as default from devise?

Reply:
Took it away.